### PR TITLE
feat: Display LXP-L balance

### PR DESCRIPTION
**Pull request by alainncls**
Displays the LXP-L balance.

image

### CODE REVIEW

File: packages/snap/src/utils.ts [hunk 1]
```diff
@@ -2,6 +2,12 @@ import { ManageStateOperation } from '@metamask/snaps-sdk';
 
 import type { Captions, SnapState } from './types';
 
+export const LXP_CONTRACT_ADDRESS =
+  '0xd83af4fbD77f3AB65C3B1Dc4B38D7e67AEcf599A';
+// TODO: change this to the correct contract address
+export const LXP_L_CONTRACT_ADDRESS =
+  '0x5FBDF89403270a1846F5ae7D113A989F850d1566';
+
```
Comment (alainncls) [inline packages/snap/src/utils.ts:10]: Update this address

File: packages/snap/src/service.ts [hunk 2]
```diff
@@ -55,9 +64,28 @@ export async function getLxpBalanceForAddress(
...
+
+/**
+ * Get the LXP-L balance for an address.
+ * @param address - The address to get the LXP-L balance for.
+ * @param chainId - The chain ID.
+ * @returns The LXP balance for the address.
+ */
+export async function getLxpLBalanceForAddress(
+  address: string,
+  chainId: string,
```
Comment (mario-christopher) [inline packages/snap/src/service.ts:80]: Lxp balances are always on Linea Mainnet- so do we need to pass this as a params between methods ?
[Not a blocker]

File: packages/snap/src/service.ts [hunk 3]
```diff
@@ -55,9 +64,28 @@ export async function getLxpBalanceForAddress(
...
+) {
+  if (!address) {
+    return 0;
+  }
+  if (chainId === '0xe708') {
```
Comment (mario-christopher) [inline packages/snap/src/service.ts:85]: I dont see this `chainId` being used at all.

File: packages/snap/src/service.ts (hunk 2 above)
Comment (alainncls) [reply packages/snap/src/service.ts:80]: Yes, but depending on the network your MM is connected to, we fetch the balance via an RPC call or the Lineascan API. That's why we need it.

File: packages/snap/src/service.ts (hunk 3 above)
Comment (alainncls) [reply packages/snap/src/service.ts:85]: Well, this is where we use it: to check the network your MM is connected to.

**Comment by alainncls** [comment]
Closed in favour of #60 to display the _minted_ and _pending_ LXP-L balances

**Pull request closed without merging**